test/lsp_test_runner.cc

```diff
@@ -547,6 +547,7 @@ TEST_CASE("LSPTest") {
         opts->ruby3KeywordArgs =
             BooleanPropertyAssertion::getValue("experimental-ruby3-keyword-args", assertions).value_or(false);
         opts->typedSuper = BooleanPropertyAssertion::getValue("typed-super", assertions).value_or(true);
+        // TODO(jez) Allow allow suppressPayloadSuperclassRedefinitionFor in a testdata test assertion?
```

Comment:
Going to punt on these two TODO for now but will leave them behind as an easy thing to grep for in the future.

Reply:
```suggestion
        // TODO(jez) Allow suppressPayloadSuperclassRedefinitionFor in a testdata test assertion?
```